src/LiveApi.js

```diff
@@ -81,7 +83,7 @@ export default class LiveApi {
     }
 
     connect(connection: ?WebSocket): void {
-        const urlPlusParams = `${this.apiUrl}?l=${this.language}&app_id=${this.appId}`;
+        const urlPlusParams = `${this.apiUrl}?l=${this.language}&app_id=${this.appId}${this.brand ? `&brand=${this.brand}` : ''}`;
```

Comment:
The question is does the backend code handle `&brand=` like when it's not passed?